perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/VosManagerBlImpl.java

```diff
@@ -527,34 +528,51 @@ public Vo getVoById(PerunSession sess, int id) throws VoNotExistsException {
 	 * @throws InternalErrorException internal error
 	 */
 	private List<RichUser> getRichUsersForMemberCandidates(PerunSession sess, List<String> attrNames, String searchString) {
-		return getRichUsersForMemberCandidates(sess, null, attrNames, searchString);
+		return getRichUsersForMemberCandidates(sess, null, attrNames, searchString, null);
 	}
 
 	/**
 	 * <p>Finds RichUsers who matches the given search string. If the given Vo is null,
 	 * they are searched in the whole Perun. If th Vo is not null, then are returned
-	 * only RichUsers who has a member inside this Vo.</p>
+	 * only RichUsers who has a member inside this Vo or who has ues in any of given ExtSources.</p>
 	 * <p>The RichUsers are returned with attributes of given names.</p>
 	 *
 	 *
 	 * @param sess session
-	 * @param vo virtual organization, users are searched only inside this vo; if is null, then in the whole Perun
+	 * @param vo virtual organization, users are searched inside this vo; if is null, then in the whole Perun
 	 * @param attrNames names of attributes that will be returned
 	 * @param searchString string used to find users
+	 * @param extSources list of extSources to possibly filter RichUsers by
```

Comment:
You do not really filter them. You are just adding new users by the list of ext sources. (if vo is not null)

Reply:
I've edited the javadoc.